code/processes/iexfeed.q

```diff
@@ -0,0 +1,3 @@
+.servers.startup[]
+.iex.callbackhandle:neg .servers.gethandlebytype[`tickerplant;`any]
+.timer.repeat[.proc.cp[];0Wp;.iex.timerperiod;(`.iex.timer;`);"Publish Feed"];
```

Comment:
For some reason, when I run this, the timer function fails the first time therefore doesn't reschedule .iex.timer.  I get: 

iexfeed|iexfeed1|ERR|timer|timer ID 9 failed with error conn. OS reports: Protocol not available.  The function will not be rescheduled

 Subsequent manual runs seem to work.  do you get the same thing?  Perhaps you could error trap the function that gets added to the timer.

Reply:
Can you add this error trap please